livekit_egress.proto

```diff
@@ -182,6 +189,27 @@ message DirectFileOutput {
   }
 }
 
+enum ImageFileSuffix {
+  IMAGE_SUFFIX_INDEX = 0;
+  IMAGE_SUFFIX_TIMESTAMP = 1;
+}
+
+message ImageOutput {
+  uint32 capture_interval = 1;              // in seconds (required)
+  int32 width = 2;                          // (required)
+  int32 height = 3;                         // (required)
+  string filename_prefix = 4;               // (optional)
+  ImageFileSuffix filename_suffix = 5;      // (optional, default INDEX)
+  ImageCodec image_codec = 6;               // (optional)
+  bool disable_manifest = 7;                // disable upload of manifest file (default false)  
```

Comment:
will images have a separate manifest too?

Reply:
Good question. Currently, we upload one (identical) manifest per output (segment + file). For consistency, images should have their own, if we want a manifest for images.

 There is a bit of a question of what a manifest for images should be. One option would be a manifest file uploaded at the end of the session with a summary of the session, similar to what we do for file/segments. More useful for images may be something closer to a m3u8 playlist: a file updated after each new image is uploaded that provides a list of all image filenames with their pts/time of the day. My initial idea when writing this proto file was the latter.

Reply:
thinking out loud. per image manifest would allow them to identify the JPEGs in near-real-time, as opposed to waiting until it's over.